home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1627,3 +2035,244 @@ components:
+      properties:
+        type:
+          $ref: '#/components/schemas/InvitationType'
+        email:
+          minLength: 5
+          maxLength: 320
+          pattern: '^([\w!#$%&''*+\/=?{|}~^-]|\.){1,60}@(?!-)(?:[a-zA-Z0-9-]{2,250}+\.)+[a-zA-Z]{2,6}$'
+          type: string
+          example: other1@gmail.com
+        status:
+          $ref: '#/components/schemas/InvitationStatus'
+    ReadCooperationInvitation:
+      allOf:
+        - $ref: '#/components/schemas/ReadInvitation'
+    ReadApartmentInvitation:
+      allOf:
+        - $ref: '#/components/schemas/ReadInvitation'
+      type: object
+      properties:
+        ownership_part:
+          type: number
+          minimum: 0.0001
+          maximum: 1.0
+          multipleOf: 1e-4
+          example: 0.6588
+
+    CreateInvitation:
+      type: object
+      discriminator:
+        propertyName: type
+        mapping:
+          base: '#/components/schemas/CreateCooperationInvitation'
+          apartment: '#/components/schemas/CreateApartmentInvitation'
+      required:
+        - email
+        - type
+      properties:
+        email:
+          minLength: 5
+          maxLength: 320
+          pattern: '^([\w!#$%&''*+\/=?{|}~^-]|\.){1,60}@(?!-)(?:[a-zA-Z0-9-]{2,250}+\.)+[a-zA-Z]{2,6}$'
+          type: string
+          example: other1@gmail.com
+        type:
+          $ref: '#/components/schemas/InvitationType'
+    CreateCooperationInvitation:
+      allOf:
+        - $ref: '#/components/schemas/CreateInvitation'
+    CreateApartmentInvitation:
+      allOf:
+        - $ref: '#/components/schemas/CreateInvitation'
+      type: object
+      required:
+        - ownership_part
+      properties:
+        ownership_part:
+          type: number
+          minimum: 0.0001
+          maximum: 1.0
+          multipleOf: 1e-4
+          example: 0.6588
+
+    UpdateInvitation:
+      type: object
+      properties:
+        status:
+          $ref: '#/components/schemas/InvitationStatus'
+    UpdateCooperationInvitation:
+      allOf:
+        - $ref: '#/components/schemas/UpdateInvitation'
+    UpdateApartmentInvitation:
+      allOf:
+        - $ref: '#/components/schemas/UpdateInvitation'
+      type: object
+      properties:
+        ownership_part:
+          type: number
+          minimum: 0.0001
+          maximum: 1.0
+          multipleOf: 1e-4
+          example: 0.6588
+
+    OwnershipStatus:
+      description: Indicates the status of an Ownership
+      type: string
+      enum: &ownership_status_values
+        - active
+        - inactive
+    ReadOwnership:
+      allOf:
+        - $ref: '#/components/schemas/BaseReadView'
+      type: object
+      properties:
+        user:
+          $ref: '#/components/schemas/ReadUser'
+        ownership_part:
+          type: number
+          minimum: 0.0001
+          maximum: 1.0
+          multipleOf: 1e-4
+          example: 0.6588
+        status:
```

Comment:
Do we really need it? I think we can go without it.

Reply:
> Do we really need it? I think we can go without it.

Proposed changes have been made